File: sqry-lang-go/tests/fixtures/go/localvars/basic.go
package localvars

// Basic local variable declaration and usage.
func basicVars() {
	x := 10
	y := x + 1
	_ = y
}

// Var declaration with type.
func varDecl() {
	var count int
	count = 42
	_ = count
}

// Multiple short var declarations.
func multiShortVar() {
	a, b := 1, 2
	c := a + b
	_ = c
}

// Parameter references.
func paramRef(name string, age int) {
	result := name
	_ = result
	_ = age
}

File: sqry-lang-typescript/tests/fixtures/localvars/basic.ts
// Basic local variable declarations and usages

function simpleVar(): void {
    let x = 10;
    let y = x + 1;
    console.log(y);
}

function constVar(): void {
    const count = 42;
    console.log(count);
}

function multipleDeclarators(): void {
    let a = 1, b = 2;
    let c = a + b;
    console.log(c);
}

function paramRef(name: string, age: number): void {
    const result = name;
    console.log(result);
    console.log(age);
}
